volume/local/local_unix.go

```diff
@@ -71,6 +70,30 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
+	r1, _ := regexp.Compile(`\baddr=*`)
+	r2, _ := regexp.Compile(`\bnfs\b`)
+	if r1.MatchString(v.opts.MountOpts) && r2.MatchString(v.opts.MountType) {
```

Comment:
Also, given that we'll still check if an addr is actually set below, checking `addr=` may not even be nescessary here (but open to suggestions)